### ISSUE TITLE
Space API v 0.13 state / open

### ISSUE THREAD

**Issue by arekkusu** [slopjong/OpenSpaceDirectory#17]
In the v0.12 specs, "open" is defined as a boolean.
In the v0.13 specs, "open" is sub-object of "state" and is defined as an Array.

Is it correct that "open" should be an array in v.013 ?
In the example (1) it appear to be a boolean. Also we have defined it as a boolean in our Json and it is validated.

Is this an error in the specs or are the spec not implemented ?
I am not very familiar with JSON, so let me know if I am missing something...

(1) http://spaceapi.net/documentation#left-tab-reference

**Comment by slopjong** [comment]
Definitely not an array. If I remember correctly this has been fixed already which got not deployed yet. Yes, it is fixed in the upstream code which previously used the type of the JSON field `type` as the SpaceAPI field type instead of considering the actual values.

```
"open":{
    "description":"... long text ...",
    "required":true,
    "type": [
        "boolean",
        "null"
   ]
}
```

You'll find the full specs [here](http://spaceapi.net/specs/0.13).

**Comment by arekkusu** [comment]
Thanks for the explanation. I was only looking at http://spaceapi.net/documentation
If I understood correctly, once fixed upstream the documentation page will be  properly generated.

And I did not know about http://spaceapi.net/specs/0.13 . Useful

**Comment by slopjong** [comment]
Yes. The documentation as well as the validator page use the same specs file. If the specs are changed both pages are _updated/synced_ automatically.

**Comment by syberphunk** [comment]
According to the documentation this is still an array? Has this been updated but not documented?

**Comment by gcarreno** [comment]
Hi there,

As of this posting (2016/07/29) http://spaceapi.net/documentation still has state.open listed as an array, but on SchemaValidator the 13.json file is listing it as boolean.
